sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 
-Nuget package Azure.Core avaliable on https://www.nuget.org/packages/Azure.Core/
+# Usage Scenarios and Samples
 
-# Hello World
+## Configuring Service Clients Using ```ClientOptions```
+Azure SDK client libraries typically expose one or more _service client_ types that 
+are the main starting points for calling corresponding Azure services. 
+You can easily find these client types as their names end with the word _Client_. 
+For example, ```BlockBlobClient``` can be used to call blob storage service, 
+and ```KeyClient``` can be used to access KeyVault service cryptographic keys. 
 
-```c#
-public async Task HelloWorld()
+These client types can be instantiated by calling a simple constructor, 
+or its overload that takes various configuration options. 
+These options are passed as a parameter that extends ```ClientOptions``` class exposed by Azure.Core.
+Various service specific options are usually added to its subclasses, but a set of SDK-wide options are 
+available directly on ```ClientOptions```.
+
+```csharp
+public void ConfigureServiceClient()
 {
-    // create http pipeline
-    var options = new HttpPipeline.Options();
-    HttpPipeline pipeline = HttpPipeline.Create(options, sdkName: "test", sdkVersion: "1.0");
-
-    // create http message
-    using (HttpMessage message = pipeline.CreateMessage(options, cancellation: default)) {
-
-        // set message URI
-        var uri = new Uri(@"https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/README.md");
-        message.SetRequestLine(HttpVerb.Get, uri);
-
-        // add headers
-        message.AddHeader("Host", uri.Host);
-
-        // send message
-        await pipeline.SendMessageAsync(message).ConfigureAwait(false);
-
-        // process response
-        Response response = message.Response;
-        if (response.Status == 200) {
-            var reader = new StreamReader(response.ContentStream);
-            string responseText = reader.ReadToEnd();
-        }
-        else throw new RequestFailedException(response);
-    }       
+    // BlobConnectionOptions inherits/extends ClientOptions
+    ClientOptions options = new BlobConnectionOptions();     
+    
+    // configure retries
+    options.RetryPolicy.MaxRetries = 5; // default is 3
+    options.RetryPolicy.Mode = RetryMode.Exponential; // default is fixed retry policy
+    options.RetryPolicy.Delay = TimeSpan.FromSeconds(1); // derfault is 0.8s
+
+    // finally create BlobContainerClient, but many Azure SDK clients will work similarly
+    var client = new BlobContainerClient(connectionString, "container", options);
 }
 ```
 
-# Other Samples
+## Accessing HTTP Response Details Using ```Response<T>```
+_Service clients_ have methods that can be used to call Azure services. 
+We refer to these client methods _service methods_.
+_Service methods_ return a shared Azure.Core type ```Response<T>``` (in rare cases its non-generic sibling ```Response```).
```

Comment:
> in rare cases its  non-generic sibling **raw** `Response`).

Reply:
I changed to "non-generic sibling untyped ```Response```"